ui-react/src/components/accounts/EditImporter.tsx

```diff
@@ -15,14 +15,23 @@ export default function EditImporter({
   handleDelete,
   importer
 }: EditImporterProps) {
-    const [loginFields, setLoginFields] = useState<Record<string, string>>(importer.loginFields || {});
+    const [loginFields, setLoginFields] = useState<Record<string, string>>(importer.loginFields || {username: '', password: '', userCode: '', id: '', num: '', card6Digits: ''});
```

Comment:
OK, I see your comment in the PR.

Are you saying this default object is not related to your other changes with the field validation?

In any way:
1. I don't like this object initializing inside a `useState`, it is too long. My common solution is to define a `const enptyLoginfFileds` on the head of the code, and use it inside the `useState`.
1. I afraid that with this approach, for each importer it will save the whole object, but with only the relevant fields not empty. Don't know if we want it. (And now, if it makes your solution more complicate, consider solving it in another PR)

Reply:
Will open a new PR regarding this.